docs/reference/release-notes/860.md

```diff
@@ -19,6 +19,137 @@ These release notes identify the new features included in 8.6, including [alpha
+
+:::note
+
+- To learn more about migrating your existing clusters to the newer model, contact your Customer Success Manager.
+- Prior to 8.6, clusters were configured by hardware size (S, M, L).
+
+:::
+
+### Automatically onboard organization users via external IDP login<span class="badge badge--long" title="This feature affects Console">Console</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2385 -->
+
+Default organizations for an external identity provider are available for organizations on an Enterprise plan.
+
+- When setting up an external identity provider, you can configure up to 10 default organizations.
+- A user logging in with the configured connection is automatically assigned to these organizations with the corresponding role.
+
+:::note
+
+You must add the following information into the ticket for the support team to configure the default organizations:
+
+- Organization Id
+- Default organization roles
+
+:::
+
+### Business Knowledge Models (BKM) in DMN <span class="badge badge--long" title="This feature affects Modeler">Modeler</span>
+
+<!-- https://github.com/camunda/product-hub/issues/1858 -->
+
+Business Knowledge Models (BKM) can now be implemented in a decision model.
+
+- Users can extract and reuse expressions in their DMN diagrams.
+- When writing an expression in a decision, the BKM name autocompletes together with the required parameters.
+
+### Deprecate zbctl and GO client <span class="badge badge--long" title="This feature affects Zeebe">Zeebe</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2438 -->
+
+The Go and zbctl clients are moved to the [Camunda Community Hub](https://github.com/camunda-community-hub) following the introduction of the Camunda 8 REST API for Process Management.
+
+- These clients are no longer officially supported by Camunda, as [announced on our blog](https://camunda.com/blog/2024/09/deprecating-zbctl-and-go-clients/).
+- Use the REST API for easier interaction with the Zeebe engine, and work with it in the same way as any other REST service.
+- Use the OpenAPI definition, the supported Postman collection, or cURL to get started implementing API requests in your code. We also recommend trying the new clients and SDKs.
+
+:::note
+Interested in maintaining these community projects? Contact us by completing the [Maintainer Proposal form](https://github.com/Camunda-Community-Hub/community/issues/new/choose) in the Camunda Community Hub.
+:::
+
+### Email Connector <span class="badge badge--long" title="This feature affects Connectors">Connectors</span>
+
+<!-- https://github.com/camunda/product-hub/issues/2430 -->
+
+The new Email Connector allows you to integrate your BPMN service with any email server using POP3, IMAP, or SMTP. You can use this Connector to automate the retrieval, deletion, search, and organization of emails directly within your processes.
+
+<!-- https://github.com/camunda/issues/issues/801 -->
```

Comment:
Same nitpick, under the heading. Also, was there no corresponding PDP issue for this in the Product Hub?

Reply:
There was one after all: https://github.com/camunda/product-hub/issues/1218 - do you want me to correct this in a PR after the release?

Reply:
Will do 👍